README.md

```diff
@@ -318,16 +319,20 @@ tag in the `pom.xml` file like the following:
 
 ## Troubleshooting
 ### Missing RHSSO admin password
-When we install more times the RHSSO instance using the same DB, the default admin password available from the 
-`credential-rhsso` Secret is not reflecting the real value pf the admin user password.
+When we install the RHSSO instance several times using the same DB, the default `admin` password available from the 
+`credential-rhsso` Secret does not reflect the real value, because in that case the DB update would fail.
```

Comment:
...the default `admin` password available from the `credential-rhsso` Secret gets overwritten with a new value and thus does not anymore fit the value expected by the already initialized DB, causing updates to fail.

Reply:
agree

Reply:
What about:
When we install the RHSSO instance several times using the same DB, a new `admin` password is generated and available from the 
`credential-rhsso` Secret, but this would not match the value already stored in the DB, causing the login to fail
because of the wrong password.